sys-libs/pam/metadata.xml

```diff
@@ -16,6 +16,9 @@
 			provided by <pkg>sys-libs/db</pkg>) installed in /usr/lib and
 			will thus not work for boot-critical services authentication.
 		</flag>
+		<flag name="docs">
+			Build and install documentation (man pages, HTML, PDF).
```

Comment:
It is a QA violation to have man pages be conditional on a USE flag at all. But certainly one named "docs", which indicates HTML and PDF documentation, not roff manuals.

Also, "doc" is the standard USE flag and loads of packages use it, whereas exactly zero of them use "docs" with an "s". So this USE flag breaks the use case of everyone who actually likes documentation.